metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -63,8 +69,44 @@ def get_session_and_host():
 
             gms_host = gms_config.server
             gms_token = gms_config.token
+            return gms_host, gms_token
         except yaml.YAMLError as exc:
             click.secho(f"{DATAHUB_CONFIG_PATH} malformatted, error: {exc}", bold=True)
+    return None, None
+
+
+def get_details_from_env():
+    gms_host: Optional[str] = None
+    try:
+        gms_host = os.environ["DATAHUB_GMS_HOST"]
+    except KeyError:
+        pass
+
+    gms_token: Optional[str] = None
+    try:
+        gms_token = os.environ["DATAHUB_GMS_TOKEN"]
+    except KeyError:
+        pass
+
+    return gms_host, gms_token
+
+
+def first_non_null(ls: List[Optional[str]]) -> Optional[str]:
+    return next((el for el in ls if el is not None), None)
+
+
+def get_session_and_host():
+    session = requests.Session()
+
+    gms_host_env, gms_token_env = get_details_from_env()
+    if not should_skip_config():
+        ensure_datahub_config()
+        gms_host_conf, gms_token_conf = get_details_from_config()
+        gms_host = first_non_null([gms_host_env, gms_host_conf])
+        gms_token = first_non_null([gms_token_env, gms_token_conf])
+    else:
+        gms_host = gms_host_env
+        gms_token = gms_token_env
```

Comment:
cen we print a more informative error message in the case that DATAHUB_SKIP_CONFIG is True but DATAHUB_GMS_HOST is not set?

Reply:
done